front/pages/w/[wId]/assistant/new.tsx

```diff
@@ -170,287 +230,225 @@ export default function AssistantNew({
-                          shallow
-                          className="cursor-pointer duration-300 hover:text-action-500 active:text-action-600"
-                        >
-                          <AssistantPreview
-                            variant="item"
-                            title={agent.name}
-                            description={agent.description}
-                            pictureUrl={agent.pictureUrl}
-                            key={agent.sId}
-                            subtitle={agent.lastAuthors?.join(", ") || ""}
-                            actions={
-                              <div className="s-flex s-justify-end">
-                                <Button
-                                  icon={ChatBubbleBottomCenterTextIcon}
-                                  label="Chat"
-                                  variant="tertiary"
-                                  size="xs"
-                                  onClick={(e) => {
-                                    // Prevent click event from propagating to parent component.
-                                    e.preventDefault();
-
-                                    setSelectedAssistant({
-                                      configurationId: agent.sId,
-                                    });
-                                    setAnimate(true);
-                                  }}
-                                />
-                              </div>
-                            }
-                          />
-                        </Link>
-                      );
-                    })}
-                  </div>
-                  <Button.List isWrapping={true}>
-                    <div className="flex flex-wrap gap-2">
-                      <Button
-                        variant="tertiary"
-                        icon={ChatBubbleBottomCenterTextIcon}
-                        label={"@help, what can I use the assistants for?"}
-                        size="xs"
-                        hasMagnifying={false}
-                        onClick={async () => {
-                          await handleOpenHelpConversation(
-                            "@help, what can I use the assistants for?"
-                          );
-                        }}
-                      />
-                      <Button
-                        variant="tertiary"
-                        icon={ChatBubbleBottomCenterTextIcon}
-                        label={"@help, what are the limitations of assistants?"}
-                        size="xs"
-                        hasMagnifying={false}
-                        onClick={async () => {
-                          await handleOpenHelpConversation(
-                            "@help, what are the limitations of assistants?"
-                          );
-                        }}
-                      />
-                    </div>
-                  </Button.List>
+      <div
+        id="assistant-new-page"
```

Comment:
Let's avoid the `new` that will field "old" in a week 😅!

Reply:
(I assume :smile: )